### PR TITLE
  Feature/client card

### PR DESCRIPTION
**Pull request by LLxD**
## Motivation

The whole prototype of the website is divided into different sections, this one is about the last part of initial section

## Proposed solution

The solution is the creation of a section component that is going to contain everything that was on the prototype.
So, based on the prototype, all stylings, texts and icons were imported and applied. I added one count animator for the "increasing count" effect too!

#### Prototype
image

#### Code 
image

### CODE REVIEW

File: src/components/ClientCard/styles.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import styled from "@emotion/styled";
+
+const style = (Component) => styled(Component)`
+  * {
+    font-family: Lato;
```
Comment (MarcosNBJ) [inline src/components/ClientCard/styles.js:5]: We are already setting the font family in the Layout component.

File: src/components/ClientCard/styles.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    line-height: 22px;
+    letter-spacing: 0em;
+    text-align: center;
+    align-items: center;
+    text-transform: uppercase;
+    color: #121212;
+  }
+
+  p {
+    font-family: Lato;
```
Comment (MarcosNBJ) [inline src/components/ClientCard/styles.js:31]: No need to set font family again here.

**Review by MarcosNBJ — CHANGES_REQUESTED** [review]

File: src/components/ClientCard/styles.js (hunk 1 above)
Comment (LLxD) [reply src/components/ClientCard/styles.js]: Fixed, removing at commit 21d4a4b

File: src/components/ClientCard/styles.js (hunk 2 above)
Comment (LLxD) [reply src/components/ClientCard/styles.js]: Fixed, removing at commit 21d4a4b

**Pull request merged**